### PR TITLE
Make bad files available in exception

### PR DESCRIPTION
**Pull request by philipbl**
I am writing a python script that checks for bitrot and emails me if it detects any. It would be nice to have the file names that were affected, so I added them to the exception.

### CODE REVIEW

File: src/bitrot.py [hunk 1]
```diff
@@ -280,9 +280,9 @@ def run(self):
 
         update_sha512_integrity()
 
-        if error_count:
+        if len(errors) > 0:
```
Comment (ambv) [inline src/bitrot.py]: `if errors:`

**Comment by ambv** [comment]
Fix that nit and I'll happily merge this. Thanks for your contribution. Add yourself to Authors in README?

**Comment by philipbl** [comment]
I updated it. Thanks!

**Comment by ambv** [comment]
Awesome 🍰

**Pull request merged**